src/SprykerEco/Shared/Econda/EcondaConstants.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\Econda;
+
+interface EcondaConstants
+{
+    const ECONDA_CSV_FOLDER_PATH = 'econda_csv_folder_path';
```

Comment:
We usually use the same value here like key: `'ECONDA_CSV_FOLDER_PATH'`
This helps the architecture sniffer to confirm that this is a valid env config const.